.github/workflows/actionlint.yml

```
@@ -10,21 +10,13 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      # Note that, because our Nix configuration provisions *both* shellcheck and actionlint,
-      # actionlint is not going to install its own shellcheck.
-      # This also makes sure that this pipeline runs using
-      # the same shellcheck as the ones in Nix shells of developers.
-      - name: Install Nix with good defaults
-        uses: input-output-hk/install-nix-action@v20
+      - uses: cachix/install-nix-action@v30
```

```suggestion
      # Note that, because our Nix configuration provisions *both* shellcheck and actionlint,
      # actionlint is not going to install its own shellcheck.
      # This also makes sure that this pipeline runs using
      # the same shellcheck as the ones in Nix shells of developers.
      - uses: cachix/install-nix-action@v30
```

Rationale: I think this is not obvious, so rather keep this info

Optional